.core-coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+omit =
+    setup.py
+    tests/*
```

Comment:
Aren't tests covered by tests themselves? What's the purpose of removing tests?

Reply:
We write tests for sample-factory code, but we do not write tests for our tests.

Reply:
Correct me if I am wrong.